templates/unbound.conf.erb

```diff
@@ -81,11 +81,19 @@ server:
 <% end -%>
 <% if @access.is_a? Array -%>
 <% @access.each do |acc| -%>
+<% if acc.chomp().split(' ').length != 1 -%>
+  access-control: <%= acc %>
+<% else -%>
```

Comment:
Could check for whitespace instead of splitting, but that's minor.  I'd think this is the kind of thing that would be suited for a test, as well as a readme update.  Readme for sure, so users who want to use it don't have to read code.